db/migrate/20180405192120_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
```

Comment:
Keep in mind this migration when you add DeviseTokenAuth to the same model. DTA's default migration will conflict and you'll have to change that one (this one is correct though).